draft-ietf-quic-recovery.md

```diff
@@ -375,7 +375,8 @@ packet, effectively restarting the connection process.
 
 Either packet indicates that the Initial was received but not processed.
 Neither packet can be treated as an acknowledgment for the Initial, but they MAY
-be used to improve the RTT estimate.
+be used to improve the RTT estimate.  Whether or not the RTT estimate is
```

Comment:
hmm, wouldn't this automatically happen with the current. Receiving a VN or Retry will send a Handshake packet. Is this just editorial clarification?

Reply:
Yes, this is intended as editorial clarification to fix #1983

Reply:
Agreed, that wasn't what I intended to say, but that's how it came out.  I'll try to fix it, or if you want to make a suggestion, that'd be great.

Reply:
Oh my suggestion was "Initial packets that are sent in response to the VN or Retry packets will reset the handshake timer."

Reply:
Thanks, done